scripts/pages/profile-actor.page.js

```diff
@@ -71,11 +97,36 @@ const _build = (data, nodeId) => {
   }
 
   if (data.top_countries.lines.length) {
-    new Top({
-      el:document.querySelector('.js-top-destination'),
-      data: data.top_countries.lines,
-      title: 'top destination countries in 2015'
-    });
+    document.querySelector('.js-top-map-title').innerHTML = `TOP DESTINATION COUNTRIES Of ${formatApostrophe(_.capitalize(data.node_name))} SOY`;
+    let topCountriesLines = data.top_countries;
+    topCountriesLines.lines = topCountriesLines.lines.slice(0, 5);
+    new Line(
+      '.js-top-destination',
+      topCountriesLines,
+      data.top_countries.includedYears,
+      {
+        margin: {top: 10, right: 100, bottom: 25, left: 94},
+        height: 244,
+        ticks: {
+          yTicks: 6,
+          yTickPadding: 10,
+          yTickFormatType: 'top-location',
+          xTickPadding: 15
+        },
+        showTooltipCallback: (country, x, y) => {
+          tooltip.showTooltip(x, y, {
+            title: `${data.node_name} > ${country.name.toUpperCase()}, ${country.date.getFullYear()}`,
+            values: [
+              { title: 'Trade Volume',
+                value: `${formatNumber(country.value)}<span>Tons</span>` }
+            ]
+          });
+        },
+        hideTooltipCallback: () => {
```

Comment:
Could the configuration of the Line component be made more DRY ? (ticks, showTooltipCallback, hideTooltipCallback look like they are the same from the other Line component)